tests/frontend/test-class-opengraph.php

```diff
@@ -401,7 +401,8 @@ public function test_image_get_content_image() {
 
 		$expected_output = '<meta property="og:image" content="' . get_home_url() . '/wp-content/plugins/wordpress-seo/tests/yoast.png" />';
 
-		$this->assertContains( $expected_output, $output );
+		// As of 7.6 we do not include external images in the og tags anymore, so this test should not contain the image.
```

Comment:
This description is causing confusion.
We are not excluding external images from the OpenGraph tags -in total-, just not accepting them from the content scraping part of the OpenGraph image fallback-mechanism.